api/v1alpha4/cluster_types.go

```diff
@@ -278,6 +278,7 @@ func (v APIEndpoint) String() string {
 // +kubebuilder:resource:path=clusters,shortName=cl,scope=Namespaced,categories=cluster-api
 // +kubebuilder:storageversion
 // +kubebuilder:subresource:status
+// +kubebuilder:printcolumn:name="Age",type="date",JSONPath=".metadata.creationTimestamp"
```

Comment:
Does this do the right thing and show the age in terms of duration from the creationTimestamp to now?

Reply:
I would also suspect that it prints the timestamp.

Interestingly there's an example in the official doc which is either fake or it works :)
https://kubernetes.io/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions/#additional-printer-columns

Reply:
Thanks for the explanation here @sbueringer I was also wondering the same thing when I saw the kubebuilder example